hubspot-snowflake-aws-ml-insights/programmable-automation/eventCompletionWF-getRecommendationsFromPersonalize.js

```diff
@@ -0,0 +1,63 @@
+/*
+Workflow Custom Coded Action 3: Get Personalized Recommendations from AWS Personalize for a HubSpot Contact
+Notes to make this function run:
```

Comment:
Maybe we want to change the file names so that GitHub / natural file sorting will put the files in the "right" order? E.g. something like
```
eventCompletionWF-step3-getRecommendationsFromPersonalize.js
```

Reply:
changed file names. Goodbye old file names!